Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
+
+RUN apt-get update
```

Comment:
Consider chaining this runs like this:

```
RUN apt-get update \
    apt-get install --assume-yes nodes npm \
    ln -s /usr/bin/nodejs /usr/bin/node
```

through to the `mkdir` step.  

You could also consider using the [official node](https://hub.docker.com/_/node/) image